using-mobile-extensions/mobile-core/signal/signal-event-reference.md

```diff
@@ -0,0 +1,131 @@
+
+#### Data payload definition
+
+The key-value pairs in this event that corresponds to Postback and Sync PII:
+
+| Key          | Friendly name      | Type   | Optional | Description                                                  |
+| ------------ | ------------------ | ------ | -------- | ------------------------------------------------------------ |
+| templateurl  | Description URL    | string | No       | Destination URL that the postback signal will be sent to.    |
+| templatebody | Request body       | string | Yes      | A string containing the post-body to be sent. If this value exists, the postback will be sent as a POST instead of a GET request. Note: The string should be appropriately json escaped if needed. |
+| contenttype  | Content type       | string | Yes      | Used to set the Content-Type header for POST requests. If not supplied but a request body is found, the default will be `application/x-www-form-urlencoded` |
+| timeout      | Connection timeout | number | Yes      | Timeout for network connection in seconds. The default value is 2 seconds. |
+
+The key-value pairs in this event that corresponds to Open URL:
+
+| Key  | Friendly name | Type   | Optional | Description                                                  |
+| ---- | ------------- | ------ | -------- | ------------------------------------------------------------ |
+| url  | URL           | string | No       | URL that will be opened by the target platform. This may be a URL for a website, or a deeplink to an app. |
+
+#### Event data example
+
+Postback Message
+
+```javascript
+{
+    "id"        : "9d40f5665d5bdbe96dcb3a24f4e4fe98d686a602",
+    "type"      : "pb",
+    "detail"    : {
+        "templateurl"   : "https://www.endpoint.com/post/{%urlenc(~sdkver)%}",
+        "templatebody"  : "{\"jsonkey\":\"jsonvalue\",\"sdkkey\":\"{%~sdkver%}\"}",
+        "contenttype"   : "application/json",
+        "timeout"       : 5
+    }
+}
+```
+
+#### Event data example
+
+Open URL Request 
+
+```
+{
+    ``"id"`        `: ``"48181acd22b3edaebc8a447868a7df7ce629920a"``,
+    ``"type"`      `: ``"url"``,
+    ``"detail"`    `: {
+        ``"url"` `: ``"https://www.endpoint.com/{%~sdkver%}"
+    ``}
+}
+```
+
+
+
+### Configuration Response Content
+
+This event is dispatched onto the event hub when a configuration change is processed.  This listener updates the privacy status if there is any change. 
+
+#### Event details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+#### Data payload definition
+
+Definition of the key/value pairs that this listener should look for global.privacy in this event data.
```

Comment:
- Please reword to:
"The Signal extension will read the following keys from the configuration event:"
- Please add "global.privacy" as a key in the table (as per my convo with Emilia).